### ISSUE TITLE
consider adding templating

### ISSUE THREAD

**Issue by gkatsev** [videojs/video.js#357]
Seems like there is a lot of string manipulation going on in the creation of the controls. Would be nicer if we were to use a templating system.
Even one as simple as:
```javascript
function template(str, arrSubs){
  var i, l;
  i = 0;
  l = arrSubs.length;
  for (; i < l; i++) {
    str = str.replace("{"+i+"}", arrSubs[i]);
  }
  return str;
}
template('foo {0} {1} {2}', [1,2,3]);
// "foo 1 2 3"
```

For example, we currently have the same string in the [`content`](https://github.com/zencoder/video-js/blob/master/src/js/controls.js#L392) and [`updateContent`](https://github.com/zencoder/video-js/blob/master/src/js/controls.js#L402) of the `durationDisplay`.
Instead, each component, could have a template associated with it that can be changed.

**Comment by mmcc** [comment]
Just to revisit this one, if we're going to use an ES6 transpiler (#1812), this is somewhat moot. Even something as simple as [string substition](http://updates.html5rocks.com/2015/01/ES6-Template-Strings) would improve things.

**Comment by mmcc** [comment]
I think we can consider this closed now that we have things like #2015 pulled in.

**Comment by gkatsev** [comment]
There is probably some extra stuff we can do to increase DRY, but yeah, we did get a lot of stuff for this via template strings.
